### PR TITLE
Add more demos to the hooman/demos folder

### PR DESCRIPTION
**Pull request by RintuNoelmon**
Hi, I have created a brick breaker game using hooman library. Please verify it.

### CODE REVIEW

File: hooman/demos/brick_breaker.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+pygame.init()
+
+brick_colors = [(255, 0, 0), (0, 255, 0), (0, 0, 255), (255, 255, 0)]  
+
+
+ball_radius = 10
+ball_x = window_width // 2
+ball_y = window_height // 2
+ball_speed_x = random.choice([-1, 1])
+ball_speed_y = -2
```
Comment (Abdur-rahmaanJ) [inline hooman/demos/brick_breaker.py:20]: Too big a value, change it to something like -0.5

File: hooman/demos/brick_breaker.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    if keys[pygame.K_RIGHT]:
+        paddle_x += paddle_speed
+
+    
+    paddle_x = max(0, min(paddle_x, window_width - paddle_width))
+
+    hapi.flip_display()
+    hapi.event_loop()
+
+pygame.quit()
```
Comment (Abdur-rahmaanJ) [inline hooman/demos/brick_breaker.py:103]: We don't need this and the next line in Hooman, you can remove them!

**Review by Abdur-rahmaanJ — CHANGES_REQUESTED** [review]

File: hooman/demos/brick_breaker.py (hunk 2 above)
Comment (RintuNoelmon) [reply hooman/demos/brick_breaker.py:103]: Thank you so much! I will make the necessary changes and raise the PR.

**Comment by RintuNoelmon** [comment]
I have made the necessary changes. Can you please verify it.

**Review by Abdur-rahmaanJ — APPROVED** [review]
Great, thanks for incorporating the feedback!

**Pull request merged**